scripts/functions/manage/truffleruby

```diff
@@ -9,6 +9,11 @@ truffleruby_post_install_hook()
 
 truffleruby_install()
 {
+  if [ "$truffleruby_version" = "23.0.0" ]; then
```

Comment:
Since the logging is meant to be a one-time heads up, this comment only applies to the `elif` logic below.

Reply:
For the `elif` in `scripts/functions/selector_interpreters`, that needs to compare exactly to 23.0.0, because for e.g. 23.1.0 or so it would be different URLs.